ai_edge_torch/convert/test/test_convert.py

```diff
@@ -306,6 +308,36 @@ def forward(self, x, y):
         model_coverage.compare_tflite_torch(edge_model, model, args_gen, kwargs_gen)
     )
 
+  def test_convert_model_with_args_nested_kwargs(self):
+    """
+    Test converting a simple model with both sample_args and sample_kwargs.
+    """
+
+    @dataclass
+    class TestContainer:
+      data_1: torch.Tensor
+      data_2: Tuple[torch.Tensor, torch.Tensor]
+
+    torch.export.register_dataclass(TestContainer, serialized_type_name="TestContainer")
```

Comment:
in the current state, wondering if we even need this line given that we flatten the args before exporting?